src/test/parse-fail/obsolete-proc.rs

```diff
@@ -10,8 +10,8 @@
 
 // Test that we generate obsolete syntax errors around usages of `proc`.
 
-fn foo(p: proc()) { } //~ ERROR obsolete syntax: the `proc` type
+fn foo(p: proc()) { } //~ ERROR: the `proc` type isn't used for the moment
```

Comment:
Ah sorry, I meant that I'm not sure that this error message is still generated. Have you run `make check-stage2-pfail` after the recent modifications?